### PR TITLE
feat: validate biome.json path config

### LINKED ISSUE #5: The `biome.json Directory Path` setting accepts file paths
It appears that the `biome.json Directory Path` settings can be set to a file path, which causes resolution problems.

Using the file picker, we're restricted to directories, but it is still possible to manually enter a path that may not be a directory path.

An example of this issue can be seen in #4, where the user uses the `/home/<...>/app/biome.json` value for the setting, which the plugin considers to be a directory, and later attempts to find `biome.json` file at `/home/<...>/app/biome.json/biome.json`

We may want to ensure that the given path is a directory.

### PR DESCRIPTION
**Pull request by victor-teles**
This PR closes #5.

- Now the field has support for directory and file.
- It now validates existence os biome.json file

### CODE REVIEW

**Review by ematipico — COMMENTED** [review]
I'm still not sure how this is going to work, but users should just pass a file name, but only the directory where the configuration is. Biome will do the rest

File: src/main/kotlin/com/github/biomejs/intellijbiome/BiomePackage.kt [hunk 1]
```diff
@@ -71,4 +72,8 @@ class BiomePackage(private val project: Project) {
             return matchResult?.value
         }.getOrNull()
     }
+
+    companion object {
+        const val configName = "biome.json"
+    }
```
[the remark this answers is not in the document]
Comment (victor-teles) [reply src/main/kotlin/com/github/biomejs/intellijbiome/BiomePackage.kt:78]: Nice, I'll update to a more flexible approach

**Comment by victor-teles** [comment]
> I'm still not sure how this is going to work, but users should just pass a file name, but only the directory where the configuration is. Biome will do the rest

You mean there's no need to validate the input?

Just to give you some context:
Before this PR, if user select the biome.json file in this field, the plugin will pass the value to CLI using --config-path and this will result in a error: `Biome couldn't read the following file, maybe for permissions reasons or it doesn't exists: /path/project/biome.json/rome.json`

To fix this now a just pass the parent path: `/path/project` instead of `/path/project/biome.json`

**Comment by ematipico** [comment]
Thank you, it wasn't clear from the PR description

**Pull request merged**